### DISCUSSION TITLE
robustData  and blockyModel

### DISCUSSION THREAD

**Discussion by makeabhishek** [gimli-org/pyGIMLi#782] [Q&A]
What is the difference between  `robustData – Robust (L1 norm mimicking) data reweighting` and  `blockyModel – Robust (L1 norm mimicking) model roughness reweighting` .

What is different mathematical formulation for each ?

**Comment by halbmy** [comment]
It is, as the description says, an L1 norm minimization (actually an L2 norm with adapted weights):
* one (robustData) for the data misfit, i.e. to reduce the effect of outliers
* the other (blockModel) for the model roughness, i.e. to enhance gradients in the model
Both can be independently set.

**Reply by makeabhishek** [reply]
Thanks for the description. Can we use both together?

Is there reference to look the mathematical description for the two conditions `robustData` and `blockModel` ?

**Comment by halbmy** [comment]
Yes, setting independently means you can set both, only one, or none.

There is nothing about L1 norm written on pygimli.org but in some textbooks or papers (Claerbout & Muir 1973).
